app/config/staging.js

```diff
@@ -15,5 +15,13 @@ module.exports = {
     secretKey: 'miniostorageengine',
     bucket: 'rra',
     region: 'us-east-1'
+  },
+  analysisProcess: {
```

Comment:
In a staging or production environment, it should really be in an environment variable.
By keeping it out of `staging.js` and `production.js`, it's harder for people to add the secret by mistake.